tests/test_vllm_client_server.py

```diff
@@ -99,6 +99,26 @@ def test_generate(self):
         for seq in completion_ids:
             assert all(isinstance(tok, int) for tok in seq)
 
+    def test_chat(self):
```

Comment:
It would be good to check that the issues with double BOS tokens getting inserted have been fully resolved (e.g. for a Llama model): https://github.com/vllm-project/vllm/issues/9519

@edbeeching ran into this during https://huggingface.co/spaces/HuggingFaceH4/blogpost-scaling-test-time-compute and it has a subtle, but negative impact on the generations.